vardict.pl

```diff
@@ -4314,7 +4341,7 @@ sub realigndel {
 	}
     }
     #@tmp = sort {$b->[2] - $b->[3] <=> $a->[2] - $a->[3]} @tmp;
-    @tmp = sort {$b->[2] <=> $a->[2]} @tmp;
+    @tmp = sort {$b->[2] <=> $a->[2] || $a->[0] <=> $b->[0] || $b->[1] cmp $a->[1] } @tmp;
```

Comment:
I propose to add an additional sort of positions and description strings, at least it will give us the same result each time. Processing order may affect the following positions. The same in realignins().